front/app/containers/HomePage/VerificationOnboardingStep.tsx

```diff
@@ -1,108 +0,0 @@
-import { Icon } from '@citizenlab/cl2-component-library';
```

Comment:
Moved inside a folder + refactored, which is why it's indicated as deleted, but it still exists.